ocaml/xapi/test_ca254515.ml

```diff
@@ -0,0 +1,98 @@
+(*
```

Comment:
We could include the function & module name in the filename to ensure people will find and extend this test.